hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FieldOperations.java

```diff
@@ -42,6 +43,11 @@ public GenericRecord readGenericRecordOrPrimitive(GenericRecord genericRecord, S
         public void writeFieldFromRecordToWriter(DefaultCompactWriter writer, GenericRecord genericRecord, String fieldName) {
             writer.writeGenericRecord(fieldName, genericRecord.getGenericRecord(fieldName));
         }
+
+        @Override
+        public String readAsJsonFormattedField(AbstractGenericRecord record, String fieldName) {
```

Comment:
Is it required for SQL compact support? Maybe it could/should be part of separate PR (there are more changes like this)?

Reply:
Removed these changes from the pr. I will send them as a separate one.